src/V2/Endpoint/Account/Mount/MountEndpoint.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace GW2Treasures\GW2Api\V2\Endpoint\Account\Mount;
+
+use GW2Treasures\GW2Api\GW2Api;
+use GW2Treasures\GW2Api\V2\Authentication\AuthenticatedEndpoint;
+use GW2Treasures\GW2Api\V2\Authentication\IAuthenticatedEndpoint;
+use GW2Treasures\GW2Api\V2\Endpoint;
+
+class MountEndpoint extends Endpoint implements IAuthenticatedEndpoint {
+    use AuthenticatedEndpoint;
+
+    public function __construct(GW2Api $api, $apiKey) {
```

Comment:
I know my spacing style I used back when I started this project is pretty uncommon, but lets keep it consistent within the project.

```suggestion
    public function __construct( GW2Api $api, $apiKey ) {
```